modules/thrift/src/thrift/server/TServerFramework.cpp

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright (c) 2006- Facebook
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <algorithm>
+#include <functional>
+#include <stdexcept>
+#include <stdint.h>
+#include <thrift/server/TServerFramework.h>
+
+namespace apache
+{
+namespace thrift
+{
+namespace server
+{
+
+// using apache::thrift::concurrency::Synchronized;
```

Comment:
I can remove it, but again, this file only exists in the Zephyr tree temporarily as a workaround for missing `std::thread` and `std::mutex` support in Zephyr. I would prefer to avoid unnecessary code churn on them if possible.